app/helpers/additional_catalog_helper.rb

```diff
@@ -3,4 +3,18 @@ module AdditionalCatalogHelper
   def purl(doc_id)
     "https://digital.library.emory.edu/purl/#{doc_id}"
   end
+
+  def field_is_for_dates?(field_name)
```

Comment:
Instead of looping over an array each time, it might be quicker to just check whether the field has the string "date" in it (e.g., `field_name.include?("date")`)

Reply:
Good point--I'll do that instead.

Reply:
Changed per request.